apps/aehttp/src/aehttp_api_handler.erl

```diff
@@ -63,8 +63,8 @@ handle_request_json(Req0, State = #state{
             Req = cowboy_req:reply(400, #{}, Body, Req1),
```

Comment:
In https://github.com/aeternity/epoch/pull/1876/files I amended this warning ^^^ to info, idea being that identified error in request shall not warn the operator (but the user, that is notified via HTTP response body).